site/content/blog/2023-09-23-postman-add-on/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Postman Add-on"
+summary: "Import Postman collections with the new Postman add-on for ZAP."
+type: post
+tags:
+- blog
+- postman
+- gsoc
+- gsoc-2023
+date: "2023-09-23"
```

Comment:
Lets publish this on Monday 25th - this gives more time for reviews and we get more engagement on weekdays.